jsonpath4k/src/commonMain/kotlin/at/asitplus/jsonpath/implementation/AntlrJsonPathSemanticAnalyzerVisitor.kt

```diff
@@ -39,29 +39,29 @@ internal class AntlrJsonPathSemanticAnalyzerVisitor(
     }
 
     override fun aggregateResult(
-        aggregate: AbstractSyntaxTree<out JsonPathExpression>?,
+        aggregate: AbstractSyntaxTree<out JsonPathExpression>,
         nextResult: AbstractSyntaxTree<out JsonPathExpression>
     ): AbstractSyntaxTree<out JsonPathExpression> {
```

Comment:
Unsure about this change, but it seems to not yield any issues

Reply:
that was pulled-in by an update to latest antlr